lib/perl/Genome/Model/PhenotypeCorrelation/Command/CaseControl/Unrelated.pm

```diff
@@ -297,7 +297,8 @@ sub _create_workflow {
                 work_dir => $vep_work_directory,
                 ensembl_annotation_build => $ensembl_annotation_build,
                 log_dir => $log_dir,
-
+                analysis_build => $self->build,
+                analysis_project => $self->build->model->analysis_projects,
```

Comment:
`Genome::Model::analysis_projects` also labeled as [is_many](https://github.com/genome/genome/blob/master/lib/perl/Genome/Model.pm#L185).  As a separate PR should we deprecate the plural form and add a singular form, etc?